flags.go

```diff
@@ -106,11 +110,27 @@ func RegisterFlags() {
 	}
 }
 
+func EnvVarFlags() {
+	var value string
+
+	for f := range rtlamrFlags {
+		value = os.Getenv("RTLAMR_" + strings.ToUpper(f));
+		if 0<len(value) {
```

Comment:
This should probably be `if value != "" {`. The docs state that looking up environment variables which are not set will return an empty string.